### ISSUE TITLE
Gitlab support

### ISSUE THREAD

**Issue by lyda** [apache/kibble-scanners#1]
I note a lack of support for Gitlab - issues and CI. Is there interest in adding that?

**Comment by Humbedooh** [comment]
Definitely, if people wish to use it! We add features as people propose them, and I think GitLab would be a very good fit here. I can look at adding GitLab support later this week, unless someone beats me to it :)

**Comment by SolarisYan** [comment]
@Humbedooh  Is gitlab supported now? If not, maybe I can research how to support gitlab, but please tell me the relevant steps first
